src/app/components/articles/article-edit/article-edit.component.ts

```diff
@@ -185,22 +188,104 @@ export class ArticleEditComponent implements OnInit, OnDestroy {
       });
   }
 
+  watchFormChanges() {
+    this.articleEditFormSubscription = this.articleEditForm.valueChanges.subscribe(() => {
+      if (this.articleEditForm.dirty) {
+        this.setEditSessionTimeout();
+        if (!this.userIsEditingArticle()) {
+          this.addUserEditingStatus();
+        }
+      }
+    });
+  }
+
   setFormData(data) {
     if (data) {
       this.articleEditForm.patchValue(data);
       this.coverImageUrl$.next(data.imageUrl);
     }
   }
 
-  watchFormChanges() {
-    this.articleEditFormSubscription = this.articleEditForm.valueChanges.subscribe(() => {
-      if (this.articleEditForm.dirty) {
-        this.setEditSessionTimeout();
-        if (!this.userIsEditingArticle()) {
-          this.addUserEditingStatus();
+  onCKEditorChanged({ event, editor }: ChangeEvent) {
```

Comment:
As our usage of CKEditor grows convoluted I'm using more and more in-line comments to keep track of the what and the why. I really would like to implement our own WYSIWYG

Reply:
You are passing in '{ event, editor }: ChangeEvent' as a parameter in onCKEditorChanged(), but you never actually read 'event'. Do you need that in there? onCKEditorChanged({ editor }: ChangeEvent) seems to work fine without 'event'.

```suggestion
  onCKEditorChanged({ editor }: ChangeEvent) {
```